assets/css/woocommerce/woocommerce.scss

```diff
@@ -1984,6 +1984,31 @@ dl.variation {
 	}
 }
 
+@media (max-width: $handheld-max) {
+	.woocommerce-MyAccount-content {
+		.woocommerce-info {
```

Comment:
I see you've gone for a "surgical" fix, fixing info notices on My account page. 

The original issue was focused on this specific instance of the bug, but I'm wondering if it's better to make a general fix that makes all notices (anywhere) work well for any length of content. We could even look at making this fix in core WooCommerce – this issue might not be unique to Storefront.

Reply:
Looking up your comment in https://github.com/woocommerce/storefront/issues/1328#issuecomment-696480929 it might be better addressing this issue directly in the WooCommerce core instead of in Storefront. I'll add a note and create an issue there.